core/includes/bootstrap.inc

```diff
@@ -2532,7 +2532,7 @@ function backdrop_set_title($title = NULL, $output = CHECK_PLAIN) {
  *   The number of random bytes to fetch and base64 encode.
  *
  * @return string
- *   The base64 encoded result will have a length of up to 4 * $byte_count.
+ *   The base64 encoded result will have a length of up to ceil($byte_count × 4 / 3).
```

Comment:
I would use the `*` operator, as what the comment shows is valid PHP code. (`$byte_count` is the function parameter.)